### DISCUSSION TITLE
R.I.P.

### DISCUSSION THREAD

**Discussion by Gumkid3** [3kh0/ext-remover#58] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
It seems like this wonderful hack has finally been patched because of the new OS update, as it seems for now, the admin extensions (which is most blocker extensions) don't seem to get removed, only the normal ones (which I think are the ones you add personally idk) so yeah, until 3kh0 finds a way to fix it, this hack doesn't work anymore.

⎧ᴿᴵᴾ⎫◟◟◟◟◟◟◟◟ ❀◟(ó ̯ ò, )

**Comment by Gumkid3** [comment]
I know many people are saying to not update ur chromebook, but if you close your chromebook, it auto updates. and you can't really have your chromebook open forever.

**Reply by md4r2** [reply]
A new version or a fix is being worked on.

**Comment by 3kh0** [comment, marked as answer]
For my model, it does not force updates, it only updates if you turn it off, like if it dies or someone ESC+reload+pwrs your chromebook. So be careful, when you close your chromebook, try to keep one tab open so it tries to prevent the update.

**Comment by capalothussein** [comment]
I did this and it's working for me but If i ctrl+shift+q+q my Chromebook or power wash it. it goes away because of the new update. is there another way of doing this?

**Comment by Gumkid3** [comment]
ok
